index.html

```diff
@@ -43,6 +43,172 @@ <h3>LET'S CONNECT</h3>
         </section>
         
     </div>
+    <section class="works-section">
+        <section class="subsection-one">
+                <div>
+                    <img src="images/Snapshoot-Portfolio.png" alt="">
+                </div>
+                <div>
+                    <h3>
+                        Tonic
+                    </h3>
+                </div>
+                <div class="list-items">
+                    <h4>CANOPY</h4>
+                    <ul class="ul-list">
+                        <li>Back End Dev</li>
+                    </ul>
+                    <ul>
+                        <li>2015</li>
+                    </ul>
+                </div>
+                <div class="paragraph">
+                    <h4>
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </h4>
+                </div>
+                <div class="btn">
+                    <button type="button" class="btn-1">html</button>
+                    <button type="button" class="btn-2">css</button>
+                    <button type="button" class="btn-3">javaScript</button>
+                </div>
```

Comment:
- [ ] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list ul and each of them a li. kindly change this.